packages/core/angular/src/wallet/wallet.store.ts

```diff
@@ -103,14 +127,18 @@ export class WalletStore extends ComponentStore<WalletState> {
 
         this.setState({
             ...initialState,
-            wallets: this._config?.wallets || [],
-            name: this._localStorage.value,
+            wallets: [],
```

Comment:
What will happen if the user doesn't call `loadWallets`? Is there any way we can get type safety that compels them to configure it correctly?

Reply:
> What will happen if the user doesn't call loadWallets?

No wallets will be shown in the modal.

> Is there any way we can get type safety that compels them to configure it correctly?

I'll give this a try!

What would be considered "correct" in this case? Calling loadWallets at least once? Have at least one wallet?

Reply:
Yeah, maybe it's a non issue and we just solve this with documentation. Worst case behavior is that they forget to initialize and have no wallets, which they'll notice in testing. Was mostly wondering if there's a way to do this in the constructor or something so that the API is obvious.

Reply:
Related: can we call it `setWallets`? `loadWallets` implies the wrong behavior I think (the adapters are already loaded)

Reply:
My reasoning behind `load` was that I was "loading the wallets into the store", I initially had `setWallets`, so I'm okay either way.

Regarding the no wallets thing, what if I make the wallets type `Wallet[] | null` and make it null by default? That way the consumers should call `setWallets` at least once, even if it's with an empty list.